task_list/app/controllers/people_controller.rb

```diff
@@ -0,0 +1,19 @@
+class PeopleController < ApplicationController
+
+  def index
+    @people = Person.all
+    @title = "Peeps"
+  end
+
+  def show
+    id = params[:id]
+    @people = Person.find(id)
+    @title = "Peep Info"
+  end
+
+  def show_tasks
+    id = params[:id]
+    @tasks = Person.find(id).tasks
+    @title = "#{Person.find(id).name}'s Tasks"
```

Comment:
You are unnecessarily retrieving the person's information twice here. Each time you use `find` you are querying the database again and as a dev you want to try to query the DB as little as possible. You should be retrieving the person using your `Person.find(id)` first, then retrieving the tasks and the name from that variable. This will ensure only a single database query.